R/AllGenerics.R

```diff
@@ -36,10 +36,12 @@ setGeneric("mzMin", function(object, ...) standardGeneric("mzMin"))
 #' @rdname hidden_aliases
 setGeneric("mzMin<-", function(object, value) standardGeneric("mzMin<-"))
 #' @rdname hidden_aliases
-setGeneric("pairs", function(object, ...) standardGeneric("pairs"))
+setGeneric("pairs", function(x, ...) standardGeneric("pairs"))
```

Comment:
This is a very good point. I was actually thinking a lot how to call that function. I don't want to use `peaks` because that's going to be confusing for users: a chromatographic peak includes signal from a retention time range - you will never have a peak consisting of a single `rt` and `intensity`. I used `pairs` here to make clear that this is a data duplet, a single retention time and intensity *pair*. I will investigate a little if I find some definition in the literature.

Reply:
What I've seen so far is that the data duplets are referred to as *data points*. Proteowizard calls them `TimeIntensityPair` - so I think with `pairs` we're not completely wrong. I would not call them `peaks`, but am open for any alternative to `pairs`. @lgatto @sneumann, any suggestion/opinion?

Reply:
Update: we'll use `as.list` instead.

Reply:
Done